pkg/workceptor/remote_work.go

```diff
@@ -219,7 +219,7 @@ func (rw *remoteUnit) startRemoteUnit(ctx context.Context, conn net.Conn, reader
 
 // cancelOrReleaseRemoteUnit makes a single attempt to cancel or release a remote unit.
 func (rw *remoteUnit) cancelOrReleaseRemoteUnit(ctx context.Context, conn net.Conn, reader *bufio.Reader,
-	release bool, force bool,
+	release bool, _ bool,
```

Comment:
Nice, this can be removed :smile: 

Also the calls to it can remove the final bool please